### ISSUE TITLE
Manjaro: OpenRA (Dune 2000) hangs on startup and can't install game music from ISO

### ISSUE THREAD

**Issue by 0xDedSec42** [OpenRA/OpenRA#19705]
Kernel: 5.10.68-1-MANJARO x86_64 bits: 64 compiler: gcc v: 11.1.0
Desktop: GNOME 40.4 tk: GTK 3.24.30 wm: gnome-shell dm: GDM 40.1
Distro: Manjaro Linux base: Arch Linux
Type: Laptop System: LENOVO product: 20F5S4C000 v: ThinkPad X260
CPU Info: Dual Core model: Intel Core i5-6300U bits: 64 type: MT MCP
Graphics Device-1: Intel Skylake GT2 [HD Graphics 520] vendor: Lenovo driver: i915

Mono: 6.12.0.122-1

OpenRA release: 20210321

```
cat debug.log
Found 8 domains for movement class 2028 on map D2k Shellmap.
Found 16 domains for movement class 1964 on map D2k Shellmap.
Found 9 domains for movement class 904 on map D2k Shellmap.
--Tick: 1 (net)
```

I tried OpenRA from the official Manjaro repository and the AppImage from GitHub. Both versions start, let me choose between Advanced and Quick Install. After that, the game hangs.

Although I have a Dune 2000 ISO after choosing it as an installation source the music doesn't seem to be installed but shortly after, the music begins playing. This issue I also had on my Windows 10 machine so it doesn't seem to be Linux related.

The game hangs completely but the music plays. After clicking a few times Linux asks me if I wanted to quit the hanging process. This happens every time I try to launch the game. After deleting .config/openra completely it allows the installation process again but with the same results.

**Comment by pchote** [comment]
This is caused by a bug in the version of Mesa shipped by arch [[mesa#4847](https://gitlab.freedesktop.org/mesa/mesa/-/issues/4847)].

Setting the environment variable `GALLIUM_THREAD=0` before launching the game will solve the hang.
There is a known issue (already fixed for the next release) in the code that detected whether the music is installed - it falsely detects missing files even when you have them all installed. If the music plays, then everything is fine.

**Comment by 0xDedSec42** [comment]
Thanks for the quick answer. OpenRA now works just fine.
